### PR TITLE
#523/feat/add frontend functionality to user country title editing

### PR DESCRIPTION
**Pull request by akskokki**
Added handlers for submitting the changed country/title data to the database.

### CODE REVIEW

File: src/components/UserList/index.test.tsx [hunk 1]
```diff
@@ -205,6 +206,46 @@ describe('User list', () => {
     expect(smallConfirmCard).toBeInTheDocument();
   });
 
+  it('pressing confirm button after changing country calls fetch update', async () => {
```
Comment (erjavaskivuori) [inline src/components/UserList/index.test.tsx:209]: Should be named like it('should call...')

File: src/components/UserList/index.test.tsx [hunk 2]
```diff
@@ -205,6 +206,46 @@ describe('User list', () => {
...
+    const smallConfirmCard = await screen.findByTestId('small-confirm-card');
+    expect(smallConfirmCard).toBeInTheDocument();
+
+    const confirmButton = await screen.findByTestId('confirm-button');
+    await userEvent.click(confirmButton);
+
+    expect(mockFetch).toHaveBeenCalledTimes(1);
+  });
+
+  it('pressing confirm button after changing title calls fetch update', async () => {
```
Comment (erjavaskivuori) [inline src/components/UserList/index.test.tsx:229]: Should be named like it('should call...')

**Review by erjavaskivuori — DISMISSED** [review]
Commented on naming tests. Otherwise looking good to me!

**Review by laurelcrelia — APPROVED** [review]
Looking good now! :+1:

**Review by laurelcrelia — APPROVED** [review]
:shipit:

**Pull request merged**